packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.html

```diff
@@ -0,0 +1,41 @@
+<template>
+  <button class="slds-button slds-button_neutral refine-button slds-is-relative" onclick={openModal}
+    title={buttonLabel}>
+    <template if:true={hasButtonLabel}> {buttonLabel} </template>
+    <template if:false={hideIcon}>
+      <lightning-icon size="x-small" icon-name="utility:filterList" alternative-text={buttonLabel}
+        class={buttonIconClass}>
+      </lightning-icon>
+    </template>
+    <template if:true={activeFiltersCount}>
+      <div
+        class="refine-button_filters-badge slds-is-absolute slds-align_absolute-center slds-text-title slds-var-p-around_xxx-small">
+        <span class="slds-truncate">{activeFiltersCount}</span>
+      </div>
+    </template>
+  </button>
+  <c-quantic-modal data-id={modalId} full-screen={fullScreen}>
+    <div
+      class="slds-var-p-vertical_medium slds-is-relative slds-align_absolute-center slds-border_bottom refine-modal__header"
+      slot="header">
+      <slot name="refine-title">
+        <div class="slds-text-title_bold">{labels.sortAndFilters}</div>
+      </slot>
+      <lightning-button class="slds-is-absolute slds-align_absolute-center slds-var-p-right_large refine-modal__action"
+        onclick={closeModal} variant="base" title="Close" icon-name="utility:close">
+      </lightning-button>
+    </div>
+    <div class="slds-align_absolute-center" slot="content">
+      <div class="refine-modal__content">
```

Comment:
Out of curiosity, why did you put this element (with the `refine-modal__content` class) inside the slot? If custom content is inserted without specifying the class, it could break the styling, no?

Could you put the `slot` attribute on this element instead?

```suggestion
      <div class="refine-modal__content" slot="content">
```